### PR TITLE
WIKI-1391 Wiki Page display - High CPU usage - infinite JS check

### PR DESCRIPTION
**Pull request by nttuyen**
- Increase the checkAnchor interval to 700ms which is still acceptable but reduce the CPU load
- Fix the issue that make additional ajax request when fresh load a wiki page

### CODE REVIEW

**Comment by thomasdelhomenie** [comment]
@nttuyen I am not sure to understand the use case behind this repeated call to checkAnchor. Cannot we get rid of that ?

**Comment by nttuyen** [comment]
@thomasdelhomenie , in wiki, we also have ajax action like other app, the issue of ajax action is that the back button of browser doesn't work, browser can't track history of ajax action. In wiki, we make the browser back button work by using hash (#) on url. 

- The checkAnchor function is to check if the hash on url is changed. I think this to make it work with all browsers. I'm not sure the "hashchange" event is support now on all supported browser of PLF

**Comment by thomasdelhomenie** [comment]
@nttuyen ok I thought it was related to Anchor macro, the name is misleading. Thanks for the explanation. The use of the event "hashchange" seems indeed the way to go, it is supported on all browsers : https://caniuse.com/#search=hashchange
The other solution is to use the History API (https://developer.mozilla.org/en-US/docs/Web/API/History_API) but it would require more work, so the haschange solution is the best one for now IMO.

**Comment by nttuyen** [comment]
@thomasdelhomenie , i'm done using haschange event to replace the checkachor interval, pls review, thanks

File: wiki-webapp/src/main/webapp/javascript/eXo/wiki/UIWikiAjaxRequest.js [hunk 1]
```diff
@@ -18,34 +18,24 @@
...
 };
 
 UIWikiAjaxRequest.prototype.init = function(actionPrefix, defaultAction) {
   this.actionPrefix = actionPrefix;
   this.defaultAction = defaultAction;
-  this.isEnableCheck = true;
-  if (this.actionPrefix && this.defaultAction) {
-    this.autoCheckAnchorId = window.setInterval(this.autoCheckAnchor, this.DEFAULT_TIMEOUT_CHECK);
-    $(window).on('unload', this.destroyAll);
+  if (!this.initialized && this.actionPrefix && this.defaultAction) {
```
Comment (thomasdelhomenie) [inline wiki-webapp/src/main/webapp/javascript/eXo/wiki/UIWikiAjaxRequest.js]: Is it necessary to manage an initialized state ? Isn't this method only called once ?

**Pull request closed without merging**